Library/Formula/dcmtk.rb

```diff
@@ -12,19 +17,22 @@ class Dcmtk < Formula
 
   option "with-docs", "Install development libraries/headers and HTML docs"
   option "with-openssl", "Configure DCMTK with support for OpenSSL"
+  option "with-libiconv", "Build with brewed libiconv. Dcmtk and system libiconv can have problems with utf8."
 
   depends_on "cmake" => :build
   depends_on "doxygen" => :build if build.with? "docs"
   depends_on "libpng"
   depends_on "libtiff"
   depends_on "openssl"
+  depends_on "homebrew/dupes/libiconv" => :optional
 
   def install
     ENV.m64 if MacOS.prefer_64_bit?
 
     args = std_cmake_args
     args << "-DDCMTK_WITH_DOXYGEN=YES" if build.with? "docs"
-    args << "-DDCMTK_WITH_OPENSSL=YES" if build.with? "openssl"
+    args << "-DDCMTK_WITH_OPENSSL=YES -DWITH_OPENSSLINC=#{Formula["openssl"].opt_prefix}" if build.with? "openssl"
```

Comment:
I can't reproduce this, either from bottle or from source.

The source install actively references the correct OpenSSL with the current formula:

```
[ 44%] Building CXX object dcmnet/libsrc/CMakeFiles/dcmnet.dir/dimdump.cc.o
cd /tmp/dcmtk20150521-5874-18qn225/dcmtk-3.6.1_20150217/build/dcmnet/libsrc && /usr/local/Library/ENV/4.3/clang++   -DDCMTK_BUILD_IN_PROGRESS -DUSE_NULL_SAFE_OFSTRING -D_BSD_COMPAT -D_BSD_SOURCE -D_OSF_SOURCE -D_REENTRANT -D_XOPEN_SOURCE_EXTENDED -Ddcmnet_EXPORTS -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -I/usr/local/include -I/usr/local/opt/openssl/include -I/usr/include/libxml2
```

What's happening for you locally?